cdk/src/functions/living-docs-dashboard-generator.ts

```diff
@@ -51,10 +53,10 @@ export function createDnsDelegationText(isRootMailConfiguredBool: boolean, dnsDo
   return dnsDelegationText;
 }
 
-function escape_string (input: string) {
+function escape_string(input: string) {
   return input
     .replace(/[\\]/g, '\\\\')
-    .replace(/[\"]/g, '\\\"')
+    .replace(/[\"]/g, '\\"')
```

Comment:
is this correct?

Reply:
It seems it is: https://github.com/prettier/prettier/issues/2789